client_comparison/mvnw

```diff
@@ -0,0 +1,259 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
tego typu rzeczy to w ignore wrzucaj

Reply:
te wszystkie takie pliki nie javowe. caly katalog .mvn wrzuc w ignore client_comparision tez. W repo to nie jest nikomu potrzebne